lib/qbo_api.rb

```diff
@@ -62,30 +56,6 @@ def connection(url: get_endpoint)
     end
   end
 
-  def create(entity, payload:, params: nil)
-    request(:post, entity: entity, path: entity_path(entity), payload: payload, params: params)
-  end
-
-  def update(entity, id:, payload:, params: nil)
-    payload.merge!(set_update(entity, id))
-    request(:post, entity: entity, path: entity_path(entity), payload: payload, params: params)
-  end
-
-  def delete(entity, id:)
-    err_msg = "Delete is only for transaction entities. Use .deactivate instead"
-    raise QboApi::NotImplementedError.new, err_msg unless is_transaction_entity?(entity)
-    path = add_params_to_path(path: entity_path(entity), params: { operation: :delete })
-    payload = set_update(entity, id)
-    request(:post, entity: entity, path: path, payload: payload)
-  end
-
-  def deactivate(entity, id:)
-    err_msg = "Deactivate is only for name list entities. Use .delete instead"
-    raise QboApi::NotImplementedError.new, err_msg unless is_name_list_entity?(entity)
-    payload = set_deactivate(entity, id)
-    request(:post, entity: entity, path: entity_path(entity), payload: payload)
-  end
-
   # https://developer.intuit.com/docs/0100_quickbooks_online/0100_essentials/0085_develop_quickbooks_apps/0004_authentication_and_authorization/oauth_management_api#/Reconnect
   def disconnect
```

Comment:
I don't know where these disconnect/reconnect methods should go. Are they part of the legacy oauth api or some 'app' api?

Reply:
`#reconnect` is part of the Oauth1 API (which we need to support through the end of 2018), and `#disconnect` is the equivalent of "revoke" in Oauth2. I think they should go in with the connection code from #70.

Reply:
so, they should go in the connection methods, maybe?

Reply:
oh, oops, you said that